core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -5988,7 +5988,8 @@ components:
         workspace:
           $ref: '#/components/schemas/WorkspaceDetails'
         workspaceSubmissionStats:
-          $ref: '#/components/schemas/WorkspaceSubmissionStats'
```

Comment:
If I include the $ref, the deprecated flag and description are ignored. Plus `workspaceSubmissionStats` is then shown in the example return value, which we don't want.